### PR TITLE
Added activities to user schema and created endpoint

### PR DESCRIPTION
**Pull request by theRoughCode**
- added activites to User schema as an array of strings (activity ids)
- added PUT endpoint to add/delete/modify user activity list
- cleaned up some backend code

### CODE REVIEW

File: models/User.js [hunk 1]
```diff
@@ -8,7 +8,8 @@ var UserSchema = new mongoose.Schema({
     min : 1,
     max : 5
   },
-  phoneNumber: String
+  phoneNumber: String,
+	activities: [String]
```
Comment (amyqiu) [inline models/User.js]: Is the indenting here a bit off?

File: models/User.js [hunk 2]
```diff
@@ -20,6 +21,7 @@ UserSchema.methods.toJSONFor = function(){
     school: this.school,
     day: this.day,
     phoneNumber: this.phoneNumber,
+		activities: this.activities
```
Comment (amyqiu) [inline models/User.js]: Same issue with indent here

**Review by amyqiu — DISMISSED** [review]
Minor formatting comments, but everything else looks good!

**Comment by theRoughCode** [comment]
@amyqiu thanks!  Yea, my editor was using tabs, so it threw the formatting off

**Comment by pratyushpal** [comment]
Looks good to me! I saw you made some formatting changes. Do you want us to follow your style?

**Review by pratyushpal — APPROVED** [review]

**Review by amyqiu — APPROVED** [review]
Fixes look good!

**Comment by theRoughCode** [comment]
@pratyushpal yea, if you're returning an error, you want to have the return keyword, else the rest of the code will still execute and potentially send to res twice.  Also, if it's a single line if statement, it makes it more compact if its on one line.  That's personal preference, but let's stick to that for now.

**Pull request merged**